scripts/images/check_model_server_health.py

```
@@ -80,7 +81,25 @@ def _wait_for_health(port: int, timeout: float) -> None:
     raise RuntimeError(f"Timed out waiting for {url}: {last_error}")
 
 
-def check_server_health(image_key: str, image_reference: str, pull: bool, timeout: float) -> None:
+def _remove_image(image_reference: str) -> None:
+    """Remove a Docker image to free disk space."""
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)